error_reporting/google/cloud/error_reporting/_gax.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""GAX wrapper for Error Reporting API requests."""
+
+from google.cloud._helpers import make_secure_channel
+from google.cloud._http import DEFAULT_USER_AGENT
+
+from google.cloud.gapic.errorreporting.v1beta1 import (
+    report_errors_service_client)
+from google.cloud.grpc.devtools.clouderrorreporting.v1beta1 import (
```

Comment:
When I `pip install gapic-google-cloud-error-reporting-v1beta1` I don't get that `proto` package anywhere. Maybe something hasn't been released or there's another dependency needed?